crates/terminal/src/terminal.rs

```diff
@@ -284,7 +284,16 @@ impl TerminalBuilder {
     ) -> Result<TerminalBuilder> {
         let pty_options = {
             let alac_shell = match shell.clone() {
-                Shell::System => None,
+                Shell::System => {
+                    // by default alacritty does not use $SHELL as the default shell,
+                    // so we make it use it here if it's specified
+                    if std::env::var("SHELL").is_ok() {
```

Comment:
Are we able to pattern match on the environment variable here to avoid the unwrap inside the condition?

```suggestion
                    if let Ok(shell) = std::env::var("SHELL") {
```

Reply:
just did! thanks for the catch